ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -12518,6 +12517,20 @@ static int zend_jit_fetch_obj(dasm_State          **Dst,
 			|	jnz >1
 			|.cold_code
 			|1:
+			|	test dword [FCARG2a + offsetof(zend_property_info, flags)], ZEND_ACC_READONLY
```

Comment:
Same question here for the x86 jit - In the `else` branch for when there is actually property type info (`if (!prop_info){...}else{...}`, when it's jumping to the label `1` for IS_REFERENCE and `<= IS_FALSE`, it seems as if FCARG2a is uninitialized (I see the FCARG2a getting initialized **after** the conditional jump

Before this PR for readonly properties, FCARG2a wasn't used in the branch for `1:`, so the lack of initialization wouldn't have mattered

```c
			} else if (flags == ZEND_FETCH_REF) {
				|	IF_TYPE dl, IS_REFERENCE, >1
				if (ce && ce->ce_flags & ZEND_ACC_IMMUTABLE) {
					|	LOAD_ADDR FCARG2a, prop_info
				} else {
```

```c
					|	cmp dl, IS_FALSE
					|	jle >1
					|.cold_code
					|1:
					if (Z_REG(prop_addr) != ZREG_FCARG1a || Z_OFFSET(prop_addr) != 0) {
						|	LOAD_ZVAL_ADDR FCARG1a, prop_addr
					}
					|	LOAD_ADDR FCARG2a, prop_info
```